src/sourmash/picklist.py

```diff
@@ -41,15 +41,17 @@ class SignaturePicklist:
     supported_coltypes = ('md5', 'md5prefix8', 'md5short',
                           'name', 'ident', 'identprefix')
 
-    def __init__(self, coltype, *, pickfile=None, column_name=None):
+    def __init__(self, coltype, *, pickfile=None, column_name=None, pickstyle='include'):
```

Comment:
Maybe using an enum here, instead of string-typing? There are examples in [signatures](https://github.com/sourmash-bio/sourmash/blob/ff75ec0490fe3974a810f73f04104be61131f3f0/src/sourmash/signature.py#L20) and [search](https://github.com/sourmash-bio/sourmash/blob/0814bcc52470425810fca4b054f9c5d8527695ae/src/sourmash/search.py#L11)

Reply:
done. thanks!